app-text/booksorg/booksorg-0.3.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A Simple Books Organizer"
+DESCRIPTION="Application to Organize with ease your pdf book"
+COPYRIGHT="2014-2018 Abou Zakaria"
+LICENSE="GNU GPL v3"
+REVISION="1"
+HOMEPAGE="http://booksorg.sourceforge.net"
+SOURCE_URI="https://downloads.sourceforge.net/booksorg/booksorg_03.tar.gz"
+SOURCE_DIR="booksorg_03"
+CHECKSUM_SHA256="dcdcce89090d1539c020412d294e2f6c0ffc84a0f4f400132f57e1c1be29d234"
+PATCHES="booksorg-0.3.patchset"
+ADDITIONAL_FILES="booksorg.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	booksorg$secondaryArchSuffix = $portVersion
+	app:BooksOrg = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGl$secondaryArchSuffix
+	lib:libpoppler$secondaryArchSuffix
+	lib:libpoppler${secondaryArchSuffix}_qt5
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5DBus$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Sql$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGl$secondaryArchSuffix
+	devel:libpoppler$secondaryArchSuffix
+	devel:libpoppler${secondaryArchSuffix}_qt5
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5DBus$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Sql$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	cmd:unzip
```

Comment:
Any reason unzip is in here? It builds fine without

Reply:
oops.. sorry no valid reason..delete that. done